aws-containers not working with JHipster 6.10.1 Monolith

##### **Overview of the issue**

So basically I am trying to deploy my monolith to aws using the `aws-containers` but it complained 


```
🔧🛠️ AWS configuring
✔ ECR Auth token has been retrieved.
Run-async async() called outside a valid run-async context, callback will be ignored.
AWS default
Warning webportal is using a cache provider, scaling will not be available. Refer to an AWS native scaling service.
 conflict ../webportal/build.gradle
? Overwrite ../webportal/build.gradle? overwrite this and all others
    force ../webportal/build.gradle
   create ../webportal/src/main/java/com/dsa/portal/bootstrap/AwsSSMConfiguration.java
   create ../webportal/src/main/resources/META-INF/spring.factories
   create ../webportal/src/main/resources/config/bootstrap-aws.yml
    force base.template.yml
    force webportal.template.yml
ls: no such file or directory: /home/johndoe/IdeaProjects/webportal/build/docker
Run-async wrapped function (sync) returned a promise but async() callback must be executed to resolve.
Starting a Gradle Daemon (subsequent builds will be faster)
✖ 
✖ FAILURE: Build failed with an exception.

* What went wrong:
Task 'bootWar' not found in root project 'webportal'. Some candidates are: 'bootJar', 'bootRun'.
✖ * Try:
Run gradlew tasks
✖ to get a list of available tasks. Run with
✖ --stacktrace
✖ option to get the stack trace. Run with --info or
✖ --debug
✖ option to get more log output. Run with --scan
✖ to get full insights.
✖ * Get more help at https://help.gradle.org
✖ 

```
##### **Motivation for or Use Case**

I am trying to leverage the existing feature to ease deployment to aws instead of manual deployment.

<!-- Explain why this is a bug for you -->

##### **Reproduce the error**

Just execute `jhipster aws-containers` on a new folder using JHipster 6.10.1

##### **Related issues**

Not sure

##### **Suggest a Fix**

Not sure

##### **JHipster Version(s)**

6.10.1


##### **JHipster configuration**

```
[INFO! Using JHipster version installed locally in current project's node_modules
INFO! Executing jhipster:info
Welcome to the JHipster Information Sub-Generator

##### **JHipster Version(s)**


webportal@0.0.1-SNAPSHOT /home/johndoe/IdeaProjects/webportal
└── generator-jhipster@6.10.1 

##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**


<details>
<summary>.yo-rc.json file</summary>
<pre>
{
  "generator-jhipster": {
    "authenticationType": "oauth2",
    "cacheProvider": "redis",
    "clientFramework": "react",
    "serverPort": "8080",
    "serviceDiscoveryType": "eureka",
    "skipUserManagement": true,
    "baseName": "webportal",
    "buildTool": "gradle",
    "databaseType": "sql",
    "devDatabaseType": "postgresql",
    "enableHibernateCache": true,
    "enableSwaggerCodegen": true,
    "enableTranslation": true,
    "jhiPrefix": "jhi",
    "languages": ["in", "en"],
    "messageBroker": false,
    "nativeLanguage": "in",
    "prodDatabaseType": "postgresql",
    "searchEngine": "elasticsearch",
    "skipClient": false,
    "testFrameworks": [],
    "websocket": false,
    "packageName": "com.dsa.portal",
    "packageFolder": "com/dsa/portal",
    "skipServer": false,
    "applicationType": "monolith",
    "clientPackageManager": "npm",
    "clientTheme": "journal",
    "clientThemeVariant": "light",
    "dtoSuffix": "DTO",
    "embeddableLaunchScript": false,
    "otherModules": [],
    "useSass": true,
    "jhipsterVersion": "6.10.1",
    "creationTimestamp": 1588422840597,
    "entitySuffix": "",
    "blueprints": [],
    "lastLiquibaseTimestamp": 1593853862000
  },
  "entities": ["News"]
}

</pre>
</details>


##### **JDL for the Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

<details>
<summary>JDL entity definitions</summary>

<pre>
entity News {
  originalArticleUrl String required unique maxlength(512),
  imageSrc String maxlength(512),
  title String required maxlength(512),
  publishedDate Instant required,
  content String maxlength(65000) required,
  articleUrl String required unique maxlength(512)
}
entity Comment {
  text String required minlength(3) maxlength(100),
  createdDate Instant
}
entity Publisher {
  name String required unique,
  logoUrl String required,
  averageRating Double max(5)
}
entity Category {
  name String required unique minlength(3)
}
entity Rating {
  score Double required min(1) max(5)
}
relationship OneToMany {
  Publisher{news} to News{publisher(name) required},
  Category{news} to News{category(name) required},
  Publisher{ratings} to Rating{publisher required}
}
relationship ManyToOne {
  Comment{user(login) required} to User,
  Comment{news required} to News,
  Rating{user(login) required} to User
}
relationship ManyToMany {
  Publisher{subscribers(login) required} to User
}

paginate News with infinite-scroll
paginate Comment, Publisher, Category, Rating with pagination
service News, Publisher, Category, Rating with serviceClass
service Comment with serviceImpl
search News, Comment, Publisher, Category, Rating with elasticsearch
filter News, Comment, Publisher, Category, Rating

</pre>
</details>
```


##### **Environment and Tools**

```
openjdk version "1.8.0_265"
OpenJDK Runtime Environment (build 1.8.0_265-8u265-b01-0ubuntu2~18.04-b01)
OpenJDK 64-Bit Server VM (build 25.265-b01, mixed mode)

git version 2.17.1

node: v13.2.0

npm: 6.14.4

yeoman: 3.1.1

Docker version 19.03.6, build 369ce74a3c

docker-compose version 1.25.4, build unknown
](url)

```
##### **Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**


##### **Browsers and Operating System**

Chrome on Ubuntu 18.04-b01

-   [v] Checking this box is mandatory (this is just to show you read everything)

<!-- Love JHipster? Please consider supporting our collective:
👉  https://opencollective.com/generator-jhipster/donate -->

--- a/generators/docker-utils.js
+++ b/generators/docker-utils.js
@@ -88,7 +88,7 @@ function checkImageExist(opts = { cwd: './', appConfig: null }) {
         this.dockerBuildCommand = './mvnw -ntp -Pprod verify jib:dockerBuild';
     } else {
         imagePath = this.destinationPath(`${opts.cwd + opts.cwd}/build/docker`);
-        this.dockerBuildCommand = './gradlew bootWar -Pprod jibDockerBuild';
+        this.dockerBuildCommand = './gradlew bootJar -Pprod jibDockerBuild';
     }
 
     if (shelljs.ls(imagePath).length === 0) {

